docs/organizer-guide/index.rst

```diff
@@ -87,6 +87,43 @@ Information about contributing interview content for the community.
 
    community-spotlight-interviewing-guide
     
+Salary Surveys
```

Comment:
This shouldn't go in the index. It should be a page on surveys.

Reply:
Dammit people, don't point out when I cut corners. :-D 
Also, yup, will fix